recipes/mobilesam/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set version = "1.0.0.20231030.12d80d4e32b277de299130d8ce28cc949fb54b6c" %}
```

```suggestion
{% set git_rev = "12d80d4e32b277de299130d8ce28cc949fb54b6c" %}
{% set version = "1.0.0.20231030." + git_rev[:8] %}
```
we can limit the sha a little bit.